instrumentation/lettuce/lettuce-5.1/src/main/java/io/opentelemetry/auto/instrumentation/lettuce/v5_1/OpenTelemetryTracing.java

```diff
@@ -98,7 +98,7 @@ public TraceContext getTraceContext() {
     private final Context context;
 
     OpenTelemetryTraceContext() {
-      this.context = Context.current();
+      context = Context.current();
```

Comment:
Please put it back :)

Reply:
Find Action removes when I hit ctrl+s.  Will do.

Reply:
I need to uncheck this box in order to revert: "Remove unnecessary this to field or method".  Should we update the contributing.md to turn this off completely?

Reply:
good question. I'm ok either way on that particular save action. @iNikem do you want to turn off that save action, or did you only want to revert here since it was unrelated to this PR (no other changes to this file)?

Reply:
I just really like `this` in constructors.

Reply:
Opened #761 to remove it from the Save Actions